sys/net/gnrc/Makefile.dep

```diff
@@ -25,7 +25,8 @@ ifneq (,$(filter gnrc_gomach,$(USEMODULE)))
 endif
 
 ifneq (,$(filter gnrc_lorawan,$(USEMODULE)))
-  USEMODULE += xtimer
+  USEMODULE += ztimer_msec
+  USEMODULE += ztimer_periph_rtt
```

Comment:
This can be removed now, `periph_rtt` will be pulled if available.

Reply:
addressed and amended directly

Reply:
squash right away!